build.rs

```diff
@@ -1170,6 +1172,13 @@ fn thread_main() {
         .size_t_is_usize(true)
         .parse_callbacks(Box::new(Callbacks));
 
+    if is_wasm_emscripten {
+        let emsdk_path = PathBuf::from(env::var("EMSDK").unwrap());
+        let emsdk_sysroot = emsdk_path.join("upstream/emscripten/cache/sysroot");
```

Comment:
`/cache/` doesn't make it sound like a permanent directory.

Maybe check if the path exists and panic with an error that tells user how to create it?